docs/snippets/common/storybook-doctor.pnpm.js.mdx

```diff
@@ -0,0 +1,3 @@
+```shell
+pnpm dlx storybook@latest doctor
```

Comment:
Why don't we use `pnpx` instead?

Reply:
That's a good question, so far we've been using `pnpm dlx` across the documentation, as it was my understanding that it was deprecated and they encourage the usage of `pnpm dlx` instead.